speech-commands/demo/index.js

```diff
@@ -172,11 +176,28 @@ enterLearnWordsButton.addEventListener('click', () => {
           exampleCanvas, spectrogram.data, spectrogram.frameSize,
           spectrogram.frameSize);
       const exampleCounts = transferRecognizer.countExamples();
+      const classNames = Object.keys(exampleCounts);
+      let minCountByClass;
```

Comment:
Instead of using classNames length to determine missing, probably better
to use transferWords to iterate through the exampleCounts, and do a reduce on the array.